transport/roundtrip_test.go

```diff
@@ -249,3 +278,78 @@ func TestSimpleRoundTrip(t *testing.T) {
 		}
 	}
 }
+
+func TestSimpleRoundTripOneway(t *testing.T) {
+	trans := httpTransport{t}
+
+	tests := []struct {
+		name           string
+		requestHeaders transport.Headers
+		requestBody    string
+	}{
+		{
+			name:           "hello world",
+			requestHeaders: transport.NewHeaders().With("foo", "bar"),
+			requestBody:    "hello world",
+		},
+		{
+			name:           "empty",
+			requestHeaders: transport.NewHeaders(),
+			requestBody:    "",
+		},
+	}
+
+	rootCtx := context.Background()
+
+	for _, tt := range tests {
```

Comment:
YARPC is Go 1.7+ now, so let's take advantage of subtests here.

https://blog.golang.org/subtests